pulumi/python/automation/main.py

```diff
@@ -82,7 +82,7 @@ def usage():
     -d, --debug        Enable debug output on all of the commands executed
     -b, --banner-type= Banner type to indicate which project is being executed (e.g. {', '.join(BANNER_TYPES)})
     -h, --help         Prints help information
-    -s, --stack        Specifies the Pulumi stack to use
+    -s, --stack=       Specifies the Pulumi stack to use
```

Comment:
This is just a documentation correction, right? So it always needed `stack=` but we didn't document it?  I know some flag parsing frameworks parse string representations of rules but my read of this code is that that is not the case.